README.md

```diff
@@ -37,6 +37,7 @@ This library currently supports the following cryptocurrencies and address forma
  - ETC (checksummed-hex)
  - RSK (checksummed-hex)
  - XDAI (checksummed-hex)
+ - VET (checksummed-hex)
```

Comment:
Can you provide reference link so that we can verify?

Reply:
@makoto  https://github.com/trustwallet/wallet-core/blob/8d3100f61e36d1e928ed1dea60ff7554bba0db16/tests/VeChain/SignerTests.cpp

Reply:
ah sorry, I missed out your initial comment